ROADMAP.md

```diff
@@ -77,30 +75,61 @@ Any dates listed below and the specific issues that will ship in a given milesto
-
   * Additional providers being incubated in https://github.com/crossplane-contrib
 
-* GitLab Auto DevOps Phase 2 - provision managed services from GitLab pipelines
+* GitLab Integration with Crossplane v1.0+
   * Currently the auto deploy app only supports PostgreSQL DBs
   * Support additional managed services from GitLab ADO pipelines
   * Add support for MySQL, Redis, Buckets, and more.
+  * Upgrade to Crossplane v1.0+ with XRDs & Compositions with default cloud service catalogs
 
 * Ease-of-use and improved experience
   * Standalone mode allowing Crossplane to run in a single container or process [#274](https://github.com/crossplane/crossplane/issues/274)
 
 ## Released
 
+### v1.1.0 Composition enhancements, provider code gen, security, and rate limiting
+
+* General
+  * Support Fs and Env ProviderConfig credential sources [crossplane-runtime#236](https://github.com/crossplane/crossplane-runtime/issues/236), [#2070](https://github.com/crossplane/crossplane/issues/2070) 
+    * AWS [provider-aws#518](https://github.com/crossplane/provider-aws/pull/518)
+    * Azure [provider-azure#218](https://github.com/crossplane/provider-azure/pull/218)
+    * GCP [provider-gcp#301](https://github.com/crossplane/provider-gcp/pull/301)
+  * [Guide for using Vault for Provider credentials](https://crossplane.io/docs/master/guides/vault-injection.html)
+  * Rate limiting support in the `ManagedReconciler` [#40](https://github.com/crossplane/crossplane-runtime/issues/40) - see related [blog post](https://danielmangum.com/posts/controller-runtime-client-go-rate-limiting/)
+
+* Composition
+  * Bi-directional patching for status [#1639](https://github.com/crossplane/crossplane/issues/1639)
+  * Support taking values from members to fill a connection secret [#1609](https://github.com/crossplane/crossplane/issues/1609)
+  * Support for required `fromFieldPath` patches [#2094](https://github.com/crossplane/crossplane/issues/2094)
+  * Support for reordering resources in a Composition template [#1909](https://github.com/crossplane/crossplane/issues/1909)
+  * Experimental `CustomComposition` [#1995](https://github.com/crossplane/crossplane/issues/1995)
+  * Deduplicate repeated patches via new `patchSets` field [#1972](https://github.com/crossplane/crossplane/issues/1972)
+
+* Providers
+  * AWS networking and VPC resources to v1beta1 [provider-aws#145](https://github.com/crossplane/provider-aws/issues/145)
+  * AWS tag update support for IAMRoles [#2118](https://github.com/crossplane/crossplane/issues/2118)
+  * AWS code generation of Crossplane Provider resources using the AWS Go Code Generation Pipeline:
+    * [Code Generation Guide](https://github.com/crossplane/provider-aws/blob/master/CODE_GENERATION.md)
+    * [Generate Crossplane resources for all services available in ACK - with v1alpha1 support](https://github.com/crossplane/crossplane/issues/1980)
+    * [Late initialization support](https://github.com/aws-controllers-k8s/code-generator/pull/9)
```

Comment:
this PR isn't merged, should we keep it here in 1.1 items, or move it to 1.2? (which I think it's actually already covered there with "Generate late-init function provider-aws#491")

Reply:
good point, it's a dup -- will remove it 👍